impls/factor/step7_quote/step7_quote.factor

```diff
@@ -11,7 +11,7 @@ SYMBOL: repl-env
 
 DEFER: EVAL
 
-GENERIC# eval-ast 1 ( ast env -- ast )
+GENERIC#: eval-ast 1 ( ast env -- ast )
```

Comment:
besides this, there's another GENERIC in line 52 which needs fixing (missing colon). I get an error:

```
Started with:
../factor/step7_quote/step7_quote.factor

52: GENERIC# apply 0 ( args fn -- maltype newenv/f )
            ^
No word named “GENERIC#” found in current vocabulary search path
(U) Quotation: [ c-to-factor
Testing cons function
TEST: '(cons 1 (list))' -> ['',(1)]
Exception: IOError(5, 'Input/output error')
Output before exception:
]
    Word: c-to-factor
(U) Quotation: [ [ (get-catchstack) push ] dip call => (get-catchstack) pop* ]
(O) Word: command-line-startup
(O) Word: run-script
(O) Word: run-file
(O) Word: parse-file
(O) Word: parse-stream
(O) Word: parse-fresh
(O) Word: (parse-lines)
(O) Word: (parse-until)
(O) Word: parse-until-step
(O) Word: no-word
(O) Word: throw-restarts
(O) Method: M\ object throw
(U) Quotation: [
        OBJ-CURRENT-THREAD special-object error-thread set-global
        current-continuation => error-continuation set-global
        [ original-error set-global ] [ rethrow ] bi
    ]
(cons 1 (list))

make: *** [Makefile:238: test^factor^step7] Error 1
```

(similarly for steps 8, 9, A)